### ISSUE TITLE
Building frontend assets fails

### ISSUE THREAD

**Issue by sheppe** [grafana/grafana#15922]
**What happened**: 
Running "yarn start" throws ts syntax error. Every other command on the building from source page, http://docs.grafana.org/project/building_from_source/, works as expected. Error:

 yarn start
yarn run v1.13.0
$ ts-node --project ./scripts/cli/tsconfig.json ./scripts/cli/index.ts core:start --watchTheme

C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:226
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
scripts/cli/utils/useSpinner.ts:7:21 - error TS2350: Only a void function can be called with the 'new' keyword.

7     const spinner = new ora(spinnerLabel);
                      ~~~~~~~~~~~~~~~~~~~~~

    at createTSError (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:226:12)
    at getOutput (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:335:40)
    at Object.compile (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:368:11)
    at Module.m._compile (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:414:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Object.require.extensions.(anonymous function) [as .ts] (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:417:12)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**What you expected to happen**:
Frontend assets to be built.

**How to reproduce it (as minimally and precisely as possible)**:
Follow the steps on the build page until you get to running "yarn start".

**Anything else we need to know?**:
All other commands on the build page instructions work, and I can start the Grafana backend, it just has no assets to serve. It seems like some kind of syntax error in the script that is run by "yarn start".

**Environment**:
- Grafana version: The version downloaded from that page today.
- Data source type & version:
- OS Grafana is installed on: Windows 10 Enterprise.
- User OS & Browser: Windows 10 Enterprise.
- Grafana plugins: None
- Others:

NPM version: 6.4.1
Node version: 10.15.3
Go version: 1.11
gcc version: 5.1.0

**Comment by sheppe** [comment]
I've already tried deleting the node_modules directory and re-ran: npm install --pure-lockfile 
No change in the error when I run yarn start, unfortunately.

**Comment by dprokop** [comment]
@sheppe thank you for reporting this. It's weird that the problem surfaced now after few weeks of using `ora`:) Fix is available in #15923, I'm gonna merge it as soon as it passes checks.

In the meantime, can you please provide mi with version of TS you have in your node_modules dir?

**Comment by sheppe** [comment]
> 
> 
> @sheppe thank you for reporting this. It's weird that the problem surfaced now after few weeks of using `ora`:) Fix is available in #15923, I'm gonna merge it as soon as it passes checks.
> 
> In the meantime, can you please provide mi with version of TS you have in your node_modules dir?

The version referenced in node_modules\typescript\package.json is 3.3.333

Thanks for the quick response.
